src/convert/replacements.ts

```diff
@@ -222,6 +222,7 @@ const getEnvValue = (env: string, allowUnset = false): string =>
  * Read the `replacement` property from sfdx-project.json
  */
 const readReplacementsFromProject = async (projectDir?: string): Promise<ReplacementConfig[]> => {
+  if (!projectDir) return []; // ? Allows tool to be used outside of a project without error.
```

Comment:
@aaron-csetter 
Some tests caught a problem with this.  Replacements support a projectDir passed explicitly or detected from the current working directory.  A try/catch approach should work.  Does this work for you?  If so, can you make this change and I'll get the tests running again?  I verified this change satisfies the tests.

```suggestion
  try {
    const proj = await SfProject.resolve(projectDir);
    const projJson = (await proj.resolveProjectConfig()) as { replacements?: ReplacementConfig[] };
    return projJson.replacements ?? [];
  } catch(e) {
    // ignore InvalidProjectWorkspaceError
  }
  return [];
```

Reply:
Thanks! Let me test this out locally rq

Reply:
That worked! I caught the error specifically such that it throw on everything else. 😄